.env

```diff
@@ -1,7 +1,9 @@
 ## Set this to true if you're using a load balancer, or set it to false if you're using seperate IPs for each service.
+## If you're using monolithic (the default), leave this set to true
 USE_GENERIC_CACHE=true
 
-## IP Address that the lancache load balancer will be running on
+## IP Address that the lancache monolithic instance will run on
+## Specify multiple IPs, space separated, if your lancache host has them - this can improve cache priming performance for some services (e.g. Steam)
 LANCACHE_IP=10.0.39.1
 
 DNS_IP=10.0.39.1
```

Comment:
perhaps clarify this is the ip to bind the dns service to, not the IP of the upstream dns server

DNS_BIND_IP maybe?

Reply:
Good point, will change.